reference/UiTPAS.v2.json

```diff
@@ -99,6 +102,13 @@
                           "price": 8,
                           "remaining": 1,
                           "type": "COUPON"
+                        },
+                        {
+                          "id": "REWARD5464",
+                          "name": "50% korting op een specifiek event",
```

Comment:
Maybe the name should also mention how many points it will cost to use? (I know this is just an example though)

Reply:
@erwin1 this specific example is not updated yet. The others like "50% korting voor 10 UiTPAS punten" look good!